docker/main/ngen/Dockerfile

```
@@ -906,8 +908,16 @@ COPY --chown=${USER} --from=build_bmi_sac_sma /dmod/ /dmod/
 COPY --chown=${USER} --from=build_bmi_snow_17 /dmod/ /dmod/
 
 USER root
-# Update path and make sure dataset directory is there
-RUN echo "export PATH=${PATH}" >> /etc/profile \
+# TODO: (later) consider something like this in the future (at least optionally) when downloading the mc client below:
+# ARG MINIO_CLIENT_RELEASE="RELEASE.2024-07-08T20-59-24Z"
+# ...
+# https://dl.min.io/client/mc/release/linux-amd64/archive/mc.${MINIO_CLIENT_RELEASE}
+
+# Setup minio client; also update path and make sure dataset directory is there
+RUN curl -L -o /dmod/bin/mc https://dl.min.io/client/mc/release/linux-amd64/mc \
+    && chmod +x /dmod/bin/mc \
+    && mkdir /dmod/.mc \
+    && echo "export PATH=${PATH}" >> /etc/profile \
```

Any reason not to do:

```suggestion
    && echo "export PATH=${PATH}"':$PATH' >> /etc/profile \
```

---

Is it really a good idea to allow the environment state after loading the global profile to be dependent on the prior environment state like this?  And aside from whether it's safe, is it actually useful in practice?

---

I must have missed the above line in my initial review:

```shell
ENV PATH="$VIRTUAL_ENV/bin:$PATH"
```

I think this is a moot point.